app.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * app.js
+ * Consolidates all of Houston to a single file
+ *
+ * @exports {Object} Helpers
+ * @exports {Object} Config
+ * @exports {Objcet} Pkg
+ * @exports {Object} Log
+ * @exports {Object} Db
+ */
+
+import Fs from 'fs'
+import Mongoose from 'mongoose'
+import Winston from 'winston'
+import Promise from 'bluebird'
+import Super from 'superagent-bluebird-promise'
+
+let app = {}
+
+// Convenience export of helpers
+export const Helpers = require('./helpers')
+
+// Application configuration loading from config.js
+try {
+  Fs.statSync('./config.js')
+} catch (err) {
+  console.log('It seems like you have not taken the time to setup Houston yet.')
+  console.log('Please use the example configuration file we have provided you')
+  console.log('to setup Houston.')
+  throw new Error('failed to setup Houston')
```

Comment:
i think a better error would be 'config.js file missing', which is more accurate.